velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -303,6 +303,32 @@ TEST_F(DateTimeFunctionsTest, unixTimestampCustomFormat) {
       unixTimestamp("2022-12-12 asd 07:45:31", "yyyy-MM-dd 'asd HH:mm:ss"));
 }
 
+TEST_F(DateTimeFunctionsTest, unixTimestampTimestampInput) {
+  const auto unixTimestamp = [&](std::optional<Timestamp> timestamp,
+                                 std::optional<StringView> formatStr) {
+    return evaluateOnce<int64_t>(
+        "unix_timestamp(c0, c1)", timestamp, formatStr);
+  };
+  EXPECT_EQ(0, unixTimestamp(Timestamp(0, 0), "yyyy-MM-dd"));
+  EXPECT_EQ(1, unixTimestamp(Timestamp(1, 990), "yyyy-MM-dd"));
+  EXPECT_EQ(61, unixTimestamp(Timestamp(61, 0), "yyyy-MM-dd"));
+}
+
+TEST_F(DateTimeFunctionsTest, unixTimestampDateInput) {
+  const auto unixTimestamp = [&](std::optional<int32_t> date,
+                                 std::optional<StringView> formatStr) {
+    return evaluateOnce<int64_t>(
+        "unix_timestamp(c0, c1)", {DATE(), VARCHAR()}, date, formatStr);
+  };
+  EXPECT_EQ(0, unixTimestamp(parseDate("1970-01-01"), "yyyy-MM-dd"));
```

Comment:
It seems the provided format string does not actually control the behavior.

Reply:
Since the 'format' parameter is ignored, can we just implement the timestamp and date functions without providing the this parameter?